fsw/src/ci_lab_version.h

```diff
@@ -36,14 +38,19 @@
 #define CI_LAB_MINOR_VERSION 3  /*!< @brief Minor version number */
 #define CI_LAB_REVISION      99 /*!< @brief Revision version number. Value of 99 indicates a development version.*/
```

Comment:
REVISION should be 0 instead of MISSION_REV being 00. Can use to set version string to development vs release.